src/set_theory/cardinal/basic.lean

```diff
@@ -1052,7 +1052,8 @@ lemma encodable_iff {α : Type u} : nonempty (encodable α) ↔ #α ≤ ℵ₀ :
 ⟨λ ⟨h⟩, ⟨(@encodable.encode' α h).trans equiv.ulift.symm.to_embedding⟩,
   λ ⟨h⟩, ⟨encodable.of_inj _ (h.trans equiv.ulift.to_embedding).injective⟩⟩
 
-@[simp] lemma mk_le_aleph_0 [encodable α] : #α ≤ ℵ₀ := encodable_iff.1 ⟨‹_›⟩
+@[simp] lemma mk_le_aleph_0 [countable α] : #α ≤ ℵ₀ :=
```

Comment:
I support this change, but I think that's out of scope.

Reply:
It's not that disruptive a change (because there's a countable instance given encodable) and there's only [one place](https://github.com/leanprover-community/mathlib/pull/15889/files#diff-88378ed03d7ffd040c975db815f3ecd6957dbef93411feafa3432f920b65ec80R45) that needs to be fixed. I updated PR description to include the change.